package.json

```diff
@@ -108,6 +109,7 @@
     "inquirer": "6.4.1",
     "jest": "24.8.0",
     "jest-diff": "24.8.0",
+    "jest-dom": "3.5.0",
```

Comment:
shouldn't this be @testing-library/jest-dom ?
It has been moved apparently: https://www.npmjs.com/package/jest-dom

Reply:
Nice catch. Apparently their documentation is not up to date yet https://testing-library.com/docs/ecosystem-jest-dom